Gir.toml

```diff
@@ -612,6 +612,45 @@ status = "generate"
     ignore = true
 
 [[object]]
+name = "Gtk.TreeModelFilter"
+status = "generate"
+    [[object.function]]
+    name = "clear_cache"
+    [[object.function]]
+    name = "get_model"
+    [[object.function]]
+    name = "refilter"
+    [[object.function]]
+    name = "set_visible_column"
+    [[object.function.parameter]]
+        name = "column"
```

Comment:
This and all before unneeded as it contains only "filter" params without any "action" params (like 'const')